config/routes.rb

```diff
@@ -722,6 +718,7 @@ def draw(routes_name)
       resources :repository_table_filters, only: %i(index show create update destroy)
       resources :repository_rows, only: %i(create show update) do
         member do
+          get :relationships
           get :assigned_task_list
           get :active_reminder_repository_cells
         end
```

Comment:
we probably want to avoid doing liner fixes to lines not affected by this implementation